dev-lang/spidermonkey/spidermonkey-128.14.0.ebuild

```
@@ -0,0 +1,488 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+FIREFOX_PATCHSET="firefox-128esr-patches-11.tar.xz"
+SPIDERMONKEY_PATCHSET="spidermonkey-128-patches-03.tar.xz"
```

```suggestion
SPIDERMONKEY_PATCHSET="spidermonkey-128-patches-04.tar.xz"
```

I've included the upstream patch in an updated SM patch set. You can use that.